pkg/base1/test-channel.ts

```diff
@@ -0,0 +1,538 @@
+import QUnit from "qunit-tests";
+
+import { transport_globals } from 'cockpit/_internal/transport';
+import { Channel, ChannelControlMessage, ChannelOptions, ChannelPayload } from 'cockpit/channel';
+
+// simple function to 'use' values (to avoid 'unused' warnings)
+function use(...args: unknown[]) {
+    console.log(args.length);
+}
+
+// Many of the functions in this file are not designed to run, but to be
+// typechecked.  Here's some assertions for the types we'll check.
+function is_str(x: string): void {
```

Comment:
Same question, could these just be `_x` instead, and `use()` dropped?

Reply:
Sure.  I'll go through and make that change everywhere.

Reply:
See comment above.